### PR TITLE
Quickinfo for valuelist

### PR DESCRIPTION
**Pull request by BerSie**
adding a new property "QuickInfo" to the Common.ValueList and Common.ValueListMapping terms - same semantics as the term Common.QuickInfo.

### CODE REVIEW

**Review by ralfhandl — APPROVED** [review]

File: vocabularies/Common.xml [hunk 1]
```diff
@@ -590,6 +590,10 @@ The minimum structure is
           <Annotation Term="Core.IsLanguageDependent" />
           <Annotation Term="Core.Description" String="Headline for value list, fallback is the label of the property or parameter" />
         </Property>
+        <Property Name="QuickInfo" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.IsLanguageDependent" />
```
Comment (ralfhandl) [inline vocabularies/Common.xml:594]:
```suggestion
          <Annotation Term="Common.Experimental" />
          <Annotation Term="Core.IsLanguageDependent" />
```

Should we make this initially experimental?

File: vocabularies/Common.xml [hunk 2]
```diff
@@ -661,6 +665,10 @@ The minimum structure is
...
```
Comment (ralfhandl) [inline vocabularies/Common.xml:669]:
```suggestion
          <Annotation Term="Core.IsLanguageDependent" />
          <Annotation Term="Common.Experimental" />
```

Should we make this initially experimental?

**Review by ralfhandl — COMMENTED** [review]
Experimental?

**Comment by BerSie** [comment]
Thanks, Ralf! I, again, forgot the experimental tag

**Review by RaizoC — APPROVED** [review]

**Review by RaizoC — APPROVED** [review]

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**